src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -91,21 +91,28 @@ class SideNav extends Component {
     }
 
     return (
-      <ul
+      <div
+        id="va-sidenav-wrapper"
         className={`usa-width-one-fourth va-sidenav vads-u-margin--1 medium-screen:vads-u-height--auto medium-screen:vads-u-margin-y--0 medium-screen:vads-u-margin-left--0 medium-screen:vads-u-margin-right--2p5 ${
           this.state.active ? `va-sidenav-height` : null
         }`}
       >
-        <span
+        <button
+          aria-describedby="va-sidenav-ul-container"
           className="vads-u-color--primary medium-screen:vads-u-display--none va-sidenav-default-trigger"
           onClick={this.toggleUlClass}
         >
           In this section <i className="fa fa-bars" />
-        </span>
-        <div className="va-sidenav-display-onclick-line medium-screen:vads-u-display--none line" />
-        {/* Render all the items recursively. */}
-        {renderChildItems(parentMostID, 1)}
-      </ul>
+        </button>
+        <ul
+          id="va-sidenav-ul-container"
```

Comment:
If we aren't using this `id` in JS and only in CSS, can we make this a class name instead?

Reply:
Sad cowboy emoji on this one - we need an id here to tie it to an aria for for 508, @kelsonic

Reply:
Oh got it! Could we add the styling to a class on this element though instead of the id in case the id is ever removed in the future or in case we want to reuse the styling? 🙂

Reply:
Yep - there aren't any styles being applied to `va-sidenav-ul-container` - doing it all on the classes, @kelsonic :+1: